src/dynsight/_internal/trajectory/trajectory.py

```diff
@@ -0,0 +1,202 @@
+from __future__ import annotations
+
+import pickle
```

Comment:
As discussed in person, we should remove pickle to ensure version compatibility of all saved things - so I think .json files.